server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -1207,6 +1209,253 @@ def _is_the_same_cred(self, stream_path: str, tsdb_connection: str) -> bool:
+                HTTPStatus.BAD_REQUEST.value,
+                reason=f"Runtime error: {mlrun.errors.err_to_str(err)}",
+            )
+        tasks: list[asyncio.Task] = []
+        model_endpoints_instructions: list[
+            tuple[
+                mlrun.common.schemas.ModelEndpoint,
+                mm_constants.ModelEndpointCreationStrategy,
+                str,
+            ]
+        ] = self._extract_model_endpoints_from_function_graph(
+            function_name=function.metadata.name,
+            function_tag=function.metadata.tag,
+            track_models=function.spec.track_models,
+            graph=function.spec.graph,
+        )
+        router_model_endpoints_instructions: list[
+            tuple[
+                mlrun.common.schemas.ModelEndpoint,
+                mm_constants.ModelEndpointCreationStrategy,
+                str,
+            ]
+        ] = []
+        for (
+            model_endpoint,
+            creation_strategy,
+            model_path,
+        ) in model_endpoints_instructions:
+            if (
+                model_endpoint.metadata.endpoint_type
+                != mm_constants.EndpointType.ROUTER
+            ):
+                tasks.append(
+                    asyncio.create_task(
+                        framework.db.session.run_async_function_with_new_db_session(
+                            func=services.api.crud.ModelEndpoints().create_model_endpoint,
+                            model_endpoint=model_endpoint,
+                            creation_strategy=creation_strategy,
+                            model_path=model_path,
+                        )
+                    )
+                )
+            else:
+                router_model_endpoints_instructions.append(
+                    (model_endpoint, creation_strategy, model_path)
+                )
+
+        created_model_endpoint = await asyncio.gather(*tasks)
+        router_tasks: list[asyncio.Task] = []
+        for (
+            model_endpoint,
+            creation_strategy,
+            model_path,
+        ) in router_model_endpoints_instructions:
+            for mep in created_model_endpoint:
+                if mep.metadata.name in model_endpoint.spec.children:
+                    model_endpoint.spec.children_uids.append(mep.metadata.uid)
+            router_tasks.append(
+                asyncio.create_task(
+                    framework.db.session.run_async_function_with_new_db_session(
+                        func=services.api.crud.ModelEndpoints().create_model_endpoint,
+                        model_endpoint=model_endpoint,
+                        creation_strategy=creation_strategy,
+                        model_path=model_path,
+                    )
+                )
+            )
+        created_model_endpoint.extend(await asyncio.gather(*router_tasks))
+
+        return created_model_endpoint
+
+    def _extract_model_endpoints_from_function_graph(
```

Comment:
I suggest to add few lines for the func desc, in particular about the response (I have no idea what is the last str value represents)